### DISCUSSION TITLE
Are there any color themes/schemes?

### DISCUSSION THREAD

**Discussion by tars0x9752** [akinomyoga/ble.sh#210] [Q&A] [answered]
Are there any kind of color themes/schemes available for ble-face other than the default color scheme? I mean something like vim's colorscheme. (btw this is not a feature request, just asking.)

**Comment by akinomyoga** [comment, marked as answer]
No, currently there are no color themes. Nevertheless, if you have any, I welcome contributions in the [`blesh-contrib`](https://github.com/akinomyoga/blesh-contrib/tree/4345a10a9484250fe9a4f83259c21e05c20a7153) repository.

**Reply by tars0x9752** [reply]
Thanks!
